test/integration/targets/mysql_db/tasks/encoding_dump_import.yml

```
@@ -0,0 +1,70 @@
+---
+
+- set_fact: 
+    latin1_file1="{{tmp_dir}}/{{file}}"
+
+- name: Deleting Latin1 encoded Database
+  mysql_db:
+    name: '{{ db_latin1_name }}'
+    state: absent
+    login_unix_socket: '{{ mysql_socket }}'
+
+- name: create Latin1 encoded database
+  mysql_db:
+    name: '{{ db_latin1_name }}'
+    state: present
+    encoding: latin1
+    login_unix_socket: '{{ mysql_socket }}'
+
+- name: create a table in Latin1 database
+  command: mysql {{ db_latin1_name }} -e 'create table testlatin1(id int, name varchar(100));'
+
+
+# Inserting a string in latin1 into table, , this string be tested later, 
+# so report any change of content in the test too
+- name: inserting data into Latin1 database
+  command: mysql {{ db_latin1_name }} -e "insert into testlatin1 value(47,'Amédée Bôlüt');"
+
+- name: selecting table
+  command: mysql {{ db_latin1_name }} -e "select * from testlatin1;"
+  register: output
+
+- name: Dumping a table in Latin1 database
+  mysql_db:
+    name: "{{ db_latin1_name }}"
+    encoding: latin1
+    target: "{{ latin1_file1 }}"
+    state: dump
+    login_unix_socket: '{{ mysql_socket }}'
+  register: dump_result
```

```suggestion
  register: dump_result
  
- assert:
    that:
    - result is changed
```
would be cool to see this for restoring too